.github/workflows/nvd.yml

```diff
@@ -2,7 +2,7 @@ name: Periodic NVD Scan
 
 on:
   schedule:
-    - cron: '0 8 * * *' # Every day at 8:00 AM
+    - cron: '0 8 * * 1-5' # Every weekday at 8:00 AM
```

Comment:
Ahh, the weekend, when no vulnerabilities are published.

Reply:
![image](https://user-images.githubusercontent.com/32022576/165552381-fd9b4fea-8008-4385-8d2c-fc4ca5c23004.png)